src/test/data/BigBed-test.js

```diff
@@ -147,6 +148,13 @@ describe('BigBed', function() {
       });
   });
 
+  it('file with 257 contigs', function () {
+    var bigBed = new BigBed('/test-data/257-contigs.bb');
+    return bigBed.contigMap.then(contigs => {
+      expect(_.keys(contigs).length).to.equal(257);
```

Comment:
Flow needs this function to be declared in lib/underscore.js. You can add it there, and use [the values declaration as an example](https://github.com/hammerlab/pileup.js/blob/master/lib/underscore.js#L47)